modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+= TinyMCE 6.1
+:navtitle: TinyMCE 6.1
+:description: Release notes for TinyMCE 6.1
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.1 was released for {enterpriseversion} and {cloudname} on Wednesday, July 13^th^, 2022. It includes {productname} 6.1 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.1:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:bug-fixes[Bug fixes]
+* xref:changes[Changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
+* xref:known-issues[Known issues]
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.1.
+
+=== A11ychecker 3.0.2
+
+The {productname} 6.1 release includes an accompanying release of the **A11ychecker** premium plugin.
+
+**A11ychecker** 3.0.2 fixes the following UI bug:
+
+The *Tools > Accessibility Checker* command opens a dialog. As such, its menu title should be *Accessibility Checker…*.
+
+The trailing ellipsis indicates the command will not execute without further user input and is required for all menu commands that present dialogs.
+
+This update corrects both the menu title and the equivalently mis-presented toolbar tool tip text. Both strings now include the required ellipsis and present as *Accessibility Checker…*, as expected.
+
+For information on the **A11ychecker** plugin, see: xref:a11ychecker.adoc[A11ychecker plugin].
+
+
+=== Comments 3.1.0
```

Comment:
This section appears to be missing quite a bit of information... here's the 3.1.0 changelog and you can see there's nothing about the fixes:
```
### Added
- Added a title to the comment sidebar panel #TINY-8770
- Added the `tinycomments_author_avatar` option to provide a custom avatar for the current author #TINY-8695
- Added new `getEventLog` API to get a log of comment changes #TINY-8769
- Added new `CommentChange` event that gets fired when any comment specific change is made #TINY-8769

### Improved
- Comments now use a provided or generated avatar instead of showing the same icon for all users #TINY-8695
- Various block elements can now be be commented on directly #TINY-8698
- Various block elements now have an outline around them to visually indicate they are part of a comment #TINY-8698

### Changed
- Save buttons for new comments and replies are now named "Comment" instead of "Save" #TINY-8696
- The CSS styles are loaded earlier than before #TINY-8710
- Replaced "Say something ..." comment placeholder text with "Add comment..." #TINY-8782

### Fixed
- The `change` event was not fired when deleting all conversations #TINY-8561
- An additional undo level was incorrectly created when the initial HTML included embedded comment data #TINY-8498
- Opening the comments sidebar via an editor command did not respect the `skip_focus` argument #TINY-8655
- Creating or editing a comment was not blocking the UI while saving the data in callback mode #TINY-8666
- A selected `hr` block could be commented on when it should not have been possible #TINY-8698
- Previous comment replies were retained in the comment placeholder after the comment had been submitted #TINY-8736
- The comment textarea will now shrink, as well as grow, in response to comment content changes #TINY-8801
```

Reply:
All of these changes now documented.